plugins/commands/scanner/src/main/kotlin/ScannerCommand.kt

```diff
@@ -115,7 +114,8 @@ class ScannerCommand : OrtCommand(
     private val scanners by option(
         "--scanners", "-s",
         help = "A comma-separated list of scanners to use.\nPossible values are: ${ScannerWrapper.ALL.keys}"
-    ).convertToScannerWrapperFactories().default(listOf(ScanCode.Factory()))
+    ).convertToScannerWrapperFactories()
+        .default(listOfNotNull(ScannerWrapper.ALL.values.singleOrNull() ?: ScannerWrapper.ALL["ScanCode"]))
```

Comment:
Would it make sense to split-up this commit into two?

Reply:
Done.